.travis.yml

```diff
@@ -12,4 +12,6 @@ install:
   - glide install
 
 script:
+  - diff -u <(echo -n) <(gofmt -d $(git ls-files | grep '.go$' | grep -v vendor))
```

Comment:
Hmm, something wrong with rebase. This change already on `master`
still show as diff on this PR.

Reply:
@chebrolus No worry.
After the rebase, your change is also in the commit list of this PR. 
Lets select rebase&merge option, then we should be good.